prefect_hightouch/api_client/api/default/get_destination.py

```diff
@@ -1,3 +1,30 @@
+"""
+This is a module containing functions, auto-generated from the 
+REST schema, but note these are **not** Prefect tasks.
+
+Example usage shown below; be sure to replace `endpoint_fn` with the desired endpoint function.
+
+```python
+from prefect_hightouch.credentials import HightouchCredentials
+from prefect_hightouch.api_client.api.default import endpoint_fn
+
+credentials = HightouchCredentials(token="my-service-token")
+client = credentials.get_client()
+result = endpoint_fn.sync(client=client)
+```
+
+The functions are described below:
+
+- `asyncio`: Like sync but async instead of blocking
+
+- `asyncio_detailed`: Like sync_detailed but async instead of blocking
+
+- `sync`: Blocking request that returns parsed data (if successful) or None
+
+- `sync_detailed`: Blocking request that always returns a Request, optionally with parsed set if the request was successful.
```

Comment:
Leaning towards being extra explicit is useful for these modules I think.

Tangential question: where do the templates for these generated files live?
```suggestion
- `asyncio`: Non-blocking request that returns parsed data (if successful) or None. Any call must be awaited.

- `asyncio_detailed`: Non-blocking request that always returns a Request, optionally with parsed set if the request was successful. Any call must be awaited.

- `sync`: Blocking request that returns parsed data (if successful) or None.

- `sync_detailed`: Blocking request that always returns a Request, optionally with parsed set if the request was successful.
```